modules/developer-cli-odo-basic-odo-cli-commands.adoc

```diff
@@ -256,16 +256,24 @@ Link adds the appropriate secret to the environment of the source component. The
 
 == list
 
-List all the components in the current application.
+List all the components in the current application and the states of the components.
 
-.Example using list
+.The states of the components
+Pushed:: a component is pushed to the cluster.
```

Comment:
I'd consider starting with a capital "A" for the first two descriptions. It's a full sentence and it looks slightly odd not capitalized since it's on the following line.